mindee/response.py

```diff
@@ -6,6 +8,44 @@
 from mindee.logger import logger
 
 
+class Job:
+    """
+    Job class for asynchronous requests.
+
+    Will hold information on the queue a document has been submitted to.
+    """
+
+    job_id: Optional[str] = None
+    """ID of the job sent by the API in response to an enqueue request."""
+    issued_at: datetime
+    """Timestamp of the request reception by the API."""
+    available_at: Optional[datetime] = None
+    """Timestamp of the request after it has been completed."""
+    status: Optional[str] = None
+    """Status of the request, as seen by the API."""
+    milli_secs_taken: int  # Check if that one is fine as a simple int
```

Comment:
```suggestion
    millisecs_taken: int  # Check if that one is fine as a simple int
```
"millisecond" is a single word